### DISCUSSION TITLE
Documentation improvement: schemaRegistry parameter when writing DgsTypeDefinitionRegistry

### DISCUSSION THREAD

**Discussion by DarthCoder117** [Netflix/dgs-framework#1719] [Ideas]
Just a minor documentation improvement suggestion, but it would be nice to mention on the Dyanic Schemas page that the `schemaRegistry` parameter can be used to retrieve the current schema when writing a `DgsTypeDefinitionRegistry`.

https://netflix.github.io/dgs/advanced/dynamic-schemas/

Currently the function is defined like this:

```java
@DgsTypeDefinitionRegistry
public TypeDefinitionRegistry registry() {
    // ...
}
```

I spent quite a bit of time searching for a way to read what's in the existing schema, and I finally realized I should check how the Relay pagination handles things. That's where I found that the function signature can actually take a `TypeDefinitionRegistry` like this:

```java
@DgsTypeDefinitionRegistry
public TypeDefinitionRegistry registry(TypeDefinitionRegistry registry) {
    // ...
}
```

I also noticed when searching for discussions that I'm not the only one that had been looking for the same information:
https://github.com/Netflix/dgs-framework/discussions/1126

**Comment by srinivasankavitha** [comment]
Thanks for opening this issue. We will fix our docs to reflect this.

**Reply by srinivasankavitha** [reply]
Addressed and merged: https://github.com/Netflix/dgs/pull/157
